docs/providers.md

```diff
@@ -59,24 +59,21 @@ to filter searches by recreation area when using GoingToCamp.
 
 The GoingToCamp Provider currently contains the following Recreation Areas:
 
--   Algonquin Highlands, Ontario, CA (#11)
--   BC Parks, British Columbia, CA (#15)
--   Gatineau Park, Ottawa-Gatineau, Ontario-Quebec, CA (#13)
--   Hamilton, Ontario, CA (#5)
+-
```

Comment:
Just the tiniest typo here with an empty line to start the bulleted list 🙃

Reply:
Your eyes are much better than mine :) 

Fixed.